packages/block-library/src/group/edit.js

```diff
@@ -46,22 +46,25 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 	return (
 		<>
 			<InspectorAdvancedControls>
-				<SelectControl
-					label={ __( 'HTML element' ) }
-					options={ [
-						{ label: __( 'Default (<div>)' ), value: 'div' },
-						{ label: '<header>', value: 'header' },
-						{ label: '<main>', value: 'main' },
-						{ label: '<section>', value: 'section' },
-						{ label: '<article>', value: 'article' },
-						{ label: '<aside>', value: 'aside' },
-						{ label: '<footer>', value: 'footer' },
-					] }
-					value={ TagName }
-					onChange={ ( value ) =>
-						setAttributes( { tagName: value } )
-					}
-				/>
+				<div className="block-library-group-block-html-element-control">
+					<SelectControl
+						label={ __( 'HTML element' ) }
+						options={ [
+							{ label: __( 'Default (<div>)' ), value: 'div' },
+							{ label: '<header>', value: 'header' },
+							{ label: '<main>', value: 'main' },
+							{ label: '<section>', value: 'section' },
+							{ label: '<article>', value: 'article' },
+							{ label: '<aside>', value: 'aside' },
+							{ label: '<footer>', value: 'footer' },
+						] }
+						value={ TagName }
+						onChange={ ( value ) =>
+							setAttributes( { tagName: value } )
+						}
+					/>
+					<HTMLElementCheckerMessage element={ TagName } />
```

Comment:
Since we do not scan/calculate if there is something wrong with the selected `tag`, should we just use the `help` property of `SelectControl` in a similar fashion with `Featured Image` [scale control](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/post-featured-image/dimension-controls.js#L110)?

In this PR the `warning` type message might not be the best thing. What do you think?

Reply:
Yup I'd be happy to change it to that if it works better. On my "Todo" list.

Reply:
OK this is now using the `help` prop of `SelectControl`. It works just as well.

RfR @ntsekouras.